test/integration/re-export-all-exports-from-page-disallowed/pages/about.js

```diff
@@ -0,0 +1,23 @@
+import fs from 'fs'
+import findUp from 'find-up'
+
+export default ({ world }) => <h1>About {world}</h1>
+
+export async function getStaticProps() {
+  const text = fs
+    .readFileSync(
+      findUp.sync('world.txt', {
+        // prevent webpack from intercepting
+        // eslint-disable-next-line no-eval
+        cwd: eval(`__dirname`),
```

Comment:
This can use `process.cwd()` so that it does not require eval

Reply:
Thanks for your review, I've just update this pull request